ksql-common/src/test/java/io/confluent/ksql/GenericRowTest.java

```diff
@@ -122,4 +128,51 @@ public void shouldPrintStructRowCorrectly() {
 
   }
 
-}
\ No newline at end of file
+  @Test
+  public void testToString() {
+      LinkedList<Object> linkedList = new LinkedList<>();
+      GenericRow genericRow = new GenericRow(linkedList);
+      linkedList.addLast( null);
+
+      assertEquals("[ null ]", genericRow.toString());
+      assertEquals(1, linkedList.size());
+  }
+
+  @Test
+  public void testGetColumnsWhenNoneAreSet() {
+      List<Object> linkedList = new LinkedList<>();
+      GenericRow genericRow = new GenericRow(linkedList);
+
+      assertEquals(0, linkedList.size());
```

Comment:
what is this statement testing? if you are testing that the constructor of `GenericRow` does not update the linkedList, then that should be a separate test and labled as so (e.g. `shouldNotModifyInputList`) though I would argue that that test is overkill